tests/SurveySimulation/test_SurveySimulation.py

```diff
@@ -416,7 +417,7 @@ def test_choose_next_target(self):
         old_sInd is none, old_sInd in sInds, old_sInd not in sInds
         """
 
-        exclude_mods = ["SS_char_only", "SS_char_only2", "SS_det_only"]
+        exclude_mods = ["SS_char_only", "SS_char_only2", "SS_det_only", "multiSS"]
```

Comment:
okay - it's fine to exclude here, but then you need to write your own specific equivalent test for your choose_next_target.